users/models.py

```diff
@@ -25,7 +25,7 @@ def send_verification_email(self, token):
         send_mail(
             'Welcome to Penny University | Verify Your Email',
             text_email,
-            'penny.university.mod@gmail.com',  # TODO: Replace email with one that has a Penny U domain
+            'invite@pennyuniversity.org',
```

Comment:
I just figured out that this formatted email works `'"Penny University" <invite@pennyuniversity.org>'`. Here's what that generates in gmail: 
![image](https://user-images.githubusercontent.com/566533/84727534-dbebd180-af54-11ea-8b67-26ffc34330e9.png)

You mind changing that?

Reply:
Oh sweet!